index.html

```diff
@@ -5,7 +5,7 @@
 <div class="top-jumbo">
   <div class="left-jumbo">
     <h1>Hyperledger Caliper</h1>
-    <h2>Blockchain performance benchmarking for Hyperledger Besu, Burrow, Fabric, Iroha & Sawtooth, and non Hyperledger Ethereum and FISCO BCOS networks</h2>
+    <h2>Blockchain performance benchmarking for Hyperledger Besu, and Fabric, and non Hyperledger Ethereum and FISCO BCOS networks</h2>
```

Comment:
I wouldn't split the enumeration based on HL. I'd use the full name everywhere, or just HL, like: HL Besu, HL Fabric, Ethereum and FISCO BCOS